packages/panels/src/Resources/Pages/ListRecords/Tab.php

```diff
@@ -68,6 +83,14 @@ public function getBadge(): ?string
         return $this->evaluate($this->badge);
     }
 
+    /**
+     * @return string | array{50: string, 100: string, 200: string, 300: string, 400: string, 500: string, 600: string, 700: string, 800: string, 900: string, 950: string} | null
+     */
+    public function getBadgeColor(): string | array | Closure | null
```

Comment:
This shouldn't ever return a closure, right?

Reply:
You're completely right... forgot this one... sorry :)

Reply:
Isn't it nicer to extract the code to a HasBadgeColor concern/trait?

Reply:
Potentially. Maybe if we use it in other places too?

Reply:
or maybe shoud have used the HasBadge in the Tab component?

Reply:
works as well... think that's the best option for now... no more duplication

Reply:
Sounds good, though that trait is from the `actions` package.
Could you move it into `support`?

Reply:
Yes did it and updated all the other references